index.js

```diff
@@ -28,9 +28,10 @@ function indentResolve(str, options) {
     return str;
 }
 
-module.exports = function(plugins, options) {
+module.exports = function(plugins, options, filterType) {
```

Comment:
However, with regards to putting the option in `options`, it could pose a problem since that object is passed directly to PostCSS.